coral/src/app/features/approvals/utils/request-status-helper.ts

```diff
@@ -1,6 +1,20 @@
 import { RequestStatus } from "src/domain/requests";
 import { ChipStatus } from "@aivenio/aquarium";
 
+type RequestStatusUIConfig = {
+  [key in RequestStatus]: {
+    chipColor: ChipStatus;
+    name: string;
+  };
+};
+
+const requestStatusUIConfig: RequestStatusUIConfig = {
+  ALL: { chipColor: "neutral", name: "All statuses" },
```

Comment:
I'm very sorry to be so nitpicky, but maybe we should keep this as the properties:

```
{ status: "neutral", text: "All statuses" }
```

So we can do this in the Tables

```
      status: ({ requestStatus }) => requestStatusUIConfig[requestStatus]
```

🙈 I'm starting to feel bad for that kind of comment, but I also feel like it will be more delightful in the end

Reply:
Then I think we should revert this comit: https://github.com/aiven/klaw/pull/680/commits/79f864ea66327a9a02d1bc873e55acac0dc360e9

Because one of the main motivation, for me, was to avoid to pick single values top pass as status config. If the usage end up being the same as if we had two separate objects (ie like this: https://github.com/aiven/klaw/pull/680/files#diff-4217fa30ad0f210208cc986555b73c2ab4d6f0858a4d6acd61b71b43acc6b2b9R48-R49), then I think that having it as a single object serves no purpose :o

And also, maybe it makes more sense to have it as two objects if we plan on using the two values (status and text) independently.

Sorry for such an annoying  kerfuffle about such a detail 😬

Reply:
No need to be sorry, that's why we do code reviews :D Will revert!